src/processors/file-handler.ts

```diff
@@ -47,25 +50,25 @@ const JSON_LOADER_OPTIONS = {
   ]
 };
 
-export function isGeoJson(json) {
+export function isGeoJson(json: any): boolean {
   // json can be feature collection
   // or single feature
   return isPlainObject(json) && (isFeature(json) || isFeatureCollection(json));
 }
 
-export function isFeature(json) {
+export function isFeature(json: any): boolean {
   return json.type === 'Feature' && json.geometry;
 }
 
-export function isFeatureCollection(json) {
+export function isFeatureCollection(json: any): boolean {
   return json.type === 'FeatureCollection' && json.features;
 }
 
-export function isRowObject(json) {
+export function isRowObject(json: any): boolean {
   return Array.isArray(json) && isPlainObject(json[0]);
 }
 
-export function isKeplerGlMap(json) {
+export function isKeplerGlMap(json: any): Boolean {
```

Comment:
```
type ValidKeplerGlMap = {
datasets: unknown
config: unknown
info: Record<string, string>
}
```

`isKeplerGlMap(json: unknown): json is ValidKeplerGlMap`

Reply:
added